### DISCUSSION TITLE
Add a shadow of a page

### DISCUSSION THREAD

**Discussion by ByronG1987** [dlemstra/Magick.NET#943] [Help]
Hello,
I use the package nuget Magick.Net-Q8-x64
I convert a pdf document to png and I would like to add a shadow of this page...
When I try to use image.Shadow... But impossible for me to add this shadow on the page (not on the every word)

Can you help me?

Regards

**Comment by dlemstra** [comment]
Can you explain in more detail what you are trying to do? Maybe add some sample images?
